### ISSUE TITLE
Create a button for the navigation bar, which redirects to the profile (overview) page

### ISSUE THREAD

**Issue by trofej** [sopra-fs22-group-18/client#70]

**Comment by awhoa** [comment]
done

**Comment by melihcatal** [comment]
@trofej @awhoa  It's not clear who completed the task? It's assigned to @awhoa however @trofej referenced a commit. Also the referenced commit seems irrelevant. Could you please inform us ?

**Comment by trofej** [comment]
Dear Melih,

There was unfortunately a misunderstanding between me and Said. My development tasks were to create a button on the navigation bar, which redirects the user to the profile page, and to create the profile page itself. Said had the development tasks to add one design task per user story (and do it) and to do the overall design of the profile editing. However, Said created also the button for the navigation bar, the profile page and the page, where the user can edit the profile.

Said implemented that and closed the development tasks, so that is why I assigned a different commit to the development task...

This is the actual commit for my development tasks: I created the profile, added the button to the navigation bar · ***@***.*** (github.com)<https://github.com/sopra-fs22-group-18/client/commit/2606c3cd71075ad53a9df9d73893d940e57a5ff0>

I discussed this already with Said... Sorry for the inconvenience, we discovered that after the TA meeting...

Kind regards

Timon Fopp
